package.json

```diff
@@ -4,48 +4,50 @@
   "description": "[![Build Status](https://travis-ci.org/atlp-rwanda/space-bn-backend.svg?branch=develop)](https://travis-ci.org/atlp-rwanda/space-bn-backend)",
   "main": "index.js",
   "dependencies": {
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "coveralls": "^3.1.0",
-    "dotenv": "^8.2.0",
-    "express": "^4.17.1",
-    "mocha-lcov-reporter": "^1.3.0",
-    "nyc": "^15.1.0",
     "@babel/cli": "^7.12.1",
     "@babel/core": "^7.12.3",
     "@babel/node": "^7.12.1",
     "@babel/plugin-transform-regenerator": "^7.12.1",
     "@babel/plugin-transform-runtime": "^7.12.1",
     "@babel/preset-env": "^7.12.1",
     "@babel/register": "^7.12.1",
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "coveralls": "^3.1.0",
+    "dotenv": "^8.2.0",
+    "ejs": "^3.1.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "pg": "^8.5.1"
+    "express": "^4.17.1",
+    "mocha-lcov-reporter": "^1.3.0",
+    "nodemon": "^2.0.6",
+    "nyc": "^15.1.0"
   },
   "devDependencies": {
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "coveralls": "^3.1.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "pg": "^8.5.1",
     "mocha": "^8.2.1",
     "eslint": "^7.13.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "chai": "^4.2.0",
-    "chai-http": "^4.3.0",
-    "coveralls": "^3.1.0",
     "mocha-lcov-reporter": "^1.3.0",
-    "nyc": "^15.1.0",
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "@babel/node": "^7.12.6",
-    "@babel/preset-env": "^7.12.1",
-    "nodemon": "^2.0.6"
+    "nyc": "^15.1.0"
   },
   "scripts": {
-    "test": " nyc --reporter=html --reporter=text mocha --require @babel/register ./src/tests/*.test.js --recursive ./src/tests/*.test.js --exit",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls",
-    "createTables": "npx sequelize db:migrate",
-    "dropTables": "npx sequelize db:migrate:undo:all",
-    "seeders": "npx sequelize-cli db:seed:all"
+    "test": "nyc --reporter=html --reporter=text mocha --timeout 150000 -r @babel/register ./src/tests/*.test.js --recursive ./src/tests/*.test.js  --exit",
+    "lint": "eslint ./",
+    "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "coverage": "nyc reporter --reporter=text --lcov | coveralls",
+    "start:dev": "nodemon --exec babel-node src/index.js",
+    "start": "npm run build && node dist/index",
+    "build": "babel src --out-dir dist",
+    "migrate": "sequelize db:migrate",
+    "dropTables": "sequelize db:migrate:undo:all",
+    "seeders": "sequelize db:seed:all"
```

Comment:
Sequelize was not installed as a dependency so this scripts would not run

Reply:
sequelize is now reinstalled.